readthedocs/projects/version_handling.py

```diff
@@ -108,7 +108,7 @@ def version_windows(versions, major=1, minor=1, point=1):
 def parse_version_failsafe(version_string):
     try:
         return Version(version_string)
-    except InvalidVersion:
+    except (InvalidVersion, UnicodeEncodeError):
```

Comment:
Do we need to catch `UnicodeEncodeError` anymore? We already fixed the main reason of getting `UnicodeEncodeError` by removing the `str()` call in `Version.identifier_friendly`. I didn't get any test failures when I removed `UnicodeEncodeError`.